ospray_modules/opendeck/ispc/cylindricStereoTracked/CylindricStereoTrackedCamera.ispc

```diff
@@ -0,0 +1,108 @@
+/* Copyright (c) 2018, EPFL/Blue Brain Project
```

Comment:
This file seems to be missing clang formatting.

Reply:
I actually wonder how the CI build passes... Maybe it's not being built, as the plugin is neither enabled by Nix, or no VRPN is found?

Reply:
It is probably not built as all the ospray_modules are OFF by default...